src/odemis/gui/comp/stream.py

```diff
@@ -1351,6 +1351,19 @@ def add_specselwidth_ctrl(self):
 
         return lbl_selection_width, sld_selection_width
 
+    @control_bookkeeper
+    def add_run_btn(self, label):
```

Comment:
Could you move it near the other similar methods. For instance just after `add_text_field()`.
Also the other functions have the parameter called "label_text", so it'd be simpler to use the same name.